games-emulation/pcsx2/pcsx2-1.3.1.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+PLOCALES="ar_SA ca_ES cs_CZ de_DE es_ES fi_FI fr_FR hr_HR hu_HU id_ID it_IT ja_JP ko_KR ms_MY nb_NO pl_PL pt_BR ru_RU sv_SE th_TH tr_TR zh_CN zh_TW"
+
+inherit cmake-utils l10n multilib toolchain-funcs wxwidgets
+
+DESCRIPTION="A PlayStation 2 emulator"
+HOMEPAGE="http://www.pcsx2.net"
+SRC_URI="https://github.com/PCSX2/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="+wxwidgets3"
+
+RDEPEND="
+	app-arch/bzip2[abi_x86_32(-)]
+	dev-libs/libaio[abi_x86_32(-)]
+	media-libs/alsa-lib[abi_x86_32(-)]
+	media-libs/libsdl[abi_x86_32(-),joystick,sound]
```

Comment:
I have to ask, because this is a common pitfall with libsdl stuff. is `joystick` and `sound` really enough? No video, opengl or other things? The problem is... libsdl is full of dlopen stuff, so you won't know until you actually emerge libsdl with ONLY `joystick` and `sound`.